mobly/controllers/android_device.py

```diff
@@ -430,6 +431,22 @@ def take_br(test_name, begin_time, ad, destination):
   utils.concurrent_exec(take_br, args)
 
 
+class BuildInfoConstants(enum.Enum):
+  """Enums for build info constants used for AndroidDevice build info."""
+
+  BUILD_ID = 'build_id'
```

Comment:
can we contain all the info needed in this dataclass so we just need to do a loop in the `build_info` property?